.github/workflows/e2e-tests.yml

```diff
@@ -3,7 +3,7 @@ name: E2E Tests
 on:
   push:
     branches:
-      - 'master'
+      - 'e2e-moar-tests'
```

Comment:
Is this right ? :)

Reply:
Same thing as https://github.com/metabase/metabase/pull/21122#discussion_r830666387